src/index.ts

```diff
@@ -57,6 +63,22 @@ async function main() {
     PUBLIC: conf.mercuryGraphQLCurrentDataPubnet,
   };
 
+  // Why does NodeJS.fetch.RequestInfo not work for URL?
+  function fetchWithTimeout(
```

Comment:
Fails after 5 seconds of waiting for Mercury graphQL response, causing a fallback to other data sources.